openmdao/solvers/ln_direct.py

```diff
@@ -8,7 +8,22 @@
 
 class DirectSolver(ScipyGMRES):
     """ OpenMDAO LinearSolver that explicitly solves the linear system using
-    linalg.solve."""
+    linalg.solve.
+
+    Options
+    -------
+    options['atol'] :  float(1e-12)
```

Comment:
The reason for the wrapper (just added it a little bit ago in the other classes) is it's the name of the options dictionary...it turns out there are some classes (e.g. external_code) in which there are multiple options dictionaries, so you wouldn't know how to invoke the option without the dict name:
In external_code, you have `options` and `fd_options` together:
```
...
fd_options['step_type'] :  str('absolute')
        Set to absolute, relative
options['check_external_outputs'] :  bool(True)
        Check that all input or output external files exist
...
```

So that was the reasoning behind that wrapper.